source/fe/mapping_q_generic.cc

```diff
@@ -2518,8 +2518,13 @@ MappingQGeneric<dim, spacedim>::transform_real_to_unit_cell(
       //   cell and that value is returned.
       // * In 3D there is no (known to the authors) exact formula, so the Newton
       //   algorithm is used.
-      const std::array<Point<spacedim>, GeometryInfo<dim>::vertices_per_cell>
-        vertices = this->get_vertices(cell);
+      const auto vertices_ = this->get_vertices(cell);
+
+      std::array<Point<spacedim>, GeometryInfo<dim>::vertices_per_cell>
+        vertices;
+      for (unsigned int i = 0; i < vertices.size(); ++i)
+        vertices[i] = vertices_[i];
```

Comment:
Given #10983, I wonder whether we still need that code from a performance point of view; as much as I like the analytical formula to invert a bilinear mapping, I think the new code mostly eliminates the bottleneck, except maybe the memory allocation that is unnecessary. There is nothing to change here for now, it might just be an opportunity to delete 200 lines of code at some point.